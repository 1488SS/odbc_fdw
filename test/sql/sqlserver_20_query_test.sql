SELECT * FROM sqlserver_test_table;
SELECT * FROM ODBCTablesList('sqlserver_fdw', 1);