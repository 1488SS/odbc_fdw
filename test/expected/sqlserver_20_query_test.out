SELECT * FROM sqlserver_test_table;
 id | name 
----+------
  1 | aaaa
(1 row)

SELECT * FROM ODBCTablesList('sqlserver_fdw', 1);
  schema   |         name         
-----------+----------------------
 fdw_tests | sqlserver_test_table
(1 row)

